src/commands/cluster-syncslots.json

```diff
@@ -0,0 +1,16 @@
+{
+    "SYNCSLOTS": {
```

Comment:
Although this is an internal command, pls update the json file including arguments and response part as well, Thanks